### ISSUE TITLE
[SCREENREADER]: Form progress bar not reporting steps

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#4679]
## Description

This issue was originally brought up with a Higher-Level Review a11y review ([ref](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3566#issuecomment-559214317)), but is an issue associated with all forms. cc @jenstrickland 

The screenreader focused on the form progress bar says "You are currently on a progress indicator" but does not report the step or progress

<img src="https://user-images.githubusercontent.com/57469/69751833-e1e7a580-111d-11ea-97f7-cee442776436.png" /> 

## Acceptance Criteria

- [ ] Progress bar reports current step out of total steps

## Steps to Recreate

1. Begin progress on any form, e.g. 
1. Using a screen reader, place focus on the form progress bar.

## Possible Fixes (optional)

The text is already located in the `aria-valuetext` of the div with a `role="progressbar"`, point to it using an `aria-describedby` (maybe?)

**Comment by 1Copenut** [comment]
Please see this design issue for related context: https://github.com/department-of-veterans-affairs/va.gov-team/issues/1145

/cc @jenstrickland

**Comment by k80bowman** [comment]
@CourtneyChambers77 could you please make sure that this issue is fixed in our [va-progress-bar](https://design.va.gov/storybook/?path=/docs/components-va-progress-bar--default) web component?

**Comment by CourtneyChambers77** [comment]
A bounded progress bar helps users understand how much time is remaining in a process. Both sighted and blind users should be informed of the intervals with the progress bar.

**Comment by CourtneyChambers77** [comment]
I have review and this component work with JAWS - passes

**Comment by k80bowman** [comment]
@CourtneyChambers77 could you please check the [va-segmented-progress-bar](https://design.va.gov/storybook/?path=/docs/components-va-segmented-progress-bar--default) as well? Sorry, I forgot that one and it seems that this is the component currently being used in the forms system.

**Comment by CourtneyChambers77** [comment]
I was to record the defect with progress bar - when using AT device JAWS is able to hear the heading level and progress at 35%

**Comment by k80bowman** [comment]
Video from @CourtneyChambers77 using NVDA. Progress is not being reported.
https://images.zenhubusercontent.com/133843125/f26362fa-285c-4bba-8983-748b0c203428/va_segmented_progress_bar.mp4

**Comment by CourtneyChambers77** [comment]
This is the update with NVDA - While testing this component the screen reader is picking up other components within the VA Components menu list https://images.zenhubusercontent.com/133843125/01d8a521-3f79-4110-942e-235c368d3676/nvda_va_segmented_progress_bar.mp4

**Comment by CourtneyChambers77** [comment]
The progress bar was tested and reads step 2 of 6 progress bar please see the file attached with NVDA https://images.zenhubusercontent.com/133843125/48b35905-fa5f-484e-b891-56887d048a0f/nvda_va_segmented_progress_bar2.mp4

**Comment by k80bowman** [comment]
@CourtneyChambers77 would you mind re-testing this component (opened in its own window) with JAWS? When you tested it before you hadn't opened the component in its own window. I want to make sure that the reason progress wasn't being reported isn't all the Storybook stuff.

**Comment by CourtneyChambers77** [comment]
Sure

**Comment by CourtneyChambers77** [comment]
@k80bowman I have added the SR with JAWS I still hear the same output as NVDA please see the update video https://images.zenhubusercontent.com/133843125/4a19dda0-69c8-42c7-9b27-8c1868cdf8c0/jaws_va_segmented_progress_bar2.mp4

**Comment by k80bowman** [comment]
@CourtneyChambers77 Thank you. It looks like NVDA is able to read the progress, but JAWS is not. Would you have any ideas about why?

**Comment by k80bowman** [comment]
We're going to make this an engineering issue now. The objective is to figure out why JAWS is not reading the correct information for the segmented progress bar and fix it. See Courtney's video above for what JAWS reads.

**Comment by pdavies88** [comment]
Aria Documentation on Progress Bar Role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role

Screen reader currently reading the aria-valuenow (which is why you hear "2 Progress Bar 2") however adding in the `aria-valuetext={label}` will solve the issue and read out the full text as intended.

**Comment by pdavies88** [comment]
@CourtneyChambers77 updates have been made feel free to retest using this link: https://60f9b557105290003b387cd5-apbervhppz.chromatic.com/iframe.html?id=components-va-segmented-progress-bar--default&viewMode=story

**Comment by CourtneyChambers77** [comment]
@pdavies88 I was able to complete the update for the progress bar with JAWS and I am hearing 2 of 6 now - Thanks, see the demo 

https://user-images.githubusercontent.com/95867638/157280556-e9574c96-cf58-4add-9ad3-1529e1eefae6.mp4
